[2020-06-19 11:21:56,114] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /index/ HTTP/1.1" 200 10132
[2020-06-19 11:27:31,805] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /send_code/ HTTP/1.1" 200 4204
[2020-06-19 11:27:50,620] [exception.py:118] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /send_code/
Traceback (most recent call last):
  File "D:\Programs\Python\Python37\Lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\Programs\Python\Python37\Lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\Programs\Python\Python37\Lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\PycharmProjects\MyPro1\login\viewas\lazy_view\user_view.py", line 95, in send_code
    res = send_ticket_by_exchangeCode(ticket, user_id, use_scope)
  File "D:\PycharmProjects\MyPro1\login\templates\admin\activities\send_code.py", line 51, in send_ticket_by_exchangeCode
    band_activity_Code(code_list[0], user_id)
  File "D:\PycharmProjects\MyPro1\login\templates\admin\activities\send_code.py", line 231, in band_activity_Code
    'nickName': get_user_info_dict(int(user_id))['nickName'],
  File "D:\PycharmProjects\MyPro1\login\templates\admin\anchor\Anchor_Operation.py", line 85, in get_user_info_dict
    if 'recomLabel' in features:
TypeError: argument of type 'NoneType' is not iterable
[2020-06-19 11:27:50,788] [basehttp.py:124] [basehttp:log_message] [ERROR]- "POST /send_code/ HTTP/1.1" 500 91975
[2020-06-19 11:29:07,023] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /index/ HTTP/1.1" 200 10132
[2020-06-19 11:38:36,899] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /index/ HTTP/1.1" 200 10132
[2020-06-19 11:38:42,864] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /send_code/ HTTP/1.1" 200 4204
